src/test/java/com/blackNecoProject/testng/commons/Utils.java

```diff
@@ -0,0 +1,83 @@
+package com.blackNecoProject.testng.commons;
+
+import org.apache.commons.lang3.RandomStringUtils;
+
+import java.sql.Timestamp;
+import java.util.Date;
+import java.util.Random;
+
+/**
+ * Holds all the functions that aren't necessarily project related.
+ */
+public class Utils {
+
+
+    /**
+     * Waits for a moment.
+     *
+     * @param millis how many milliseconds to sleep.
+     */
+    public static void sleep(long millis){
```

Comment:
for why this class?

Reply:
Waits for some action before download or upload file to web site

Reply:
for waiting uploading or downloading some element from website